Sources/Sentry/SentrySDK.m

```diff
@@ -240,6 +240,7 @@ + (void)startWithOptions:(SentryOptions *)options
     SentryClient *newClient = [[SentryClient alloc] initWithOptions:options];
     [newClient.fileManager moveAppStateToPreviousAppState];
     [newClient.fileManager moveBreadcrumbsToPreviousBreadcrumbs];
+    [newClient.fileManager moveContextFileToPreviousContextFile];
```

Comment:
`m`: It would be valuable to add a test in SentrySDKTests to ensure we're actually calling this here.